File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/MainObject.java
package com.example.weatherchallengemvvmkotlin.data.models;

import com.google.gson.annotations.SerializedName;

public class MainObject {

    @SerializedName("temp")
    double temp;

    @SerializedName("pressure")
    double pressure;

    @SerializedName("humidity")
    long humidity;

    @SerializedName("temp_min")
    double tempMin;

    @SerializedName("temp_max")
    double tempMax;

    @SerializedName("sea_level")
    double seaLevel;

    @SerializedName("grnd_level")
    double groundLevel;

    public double getTemp() {
        return temp;
    }

    public double getPressure() {
        return pressure;
    }

    public long getHumidity() {
        return humidity;
    }

    public double getTempMin() {
        return tempMin;
    }

    public double getTempMax() {
        return tempMax;
    }

    public double getSeaLevel() {
        return seaLevel;
    }

    public double getGroundLevel() {
        return groundLevel;
    }

}

File: app/src/main/java/com/example/weatherchallengemvvmkotlin/data/models/MainObject.kt
package com.example.weatherchallengemvvmkotlin.data.models

import com.google.gson.annotations.SerializedName

class MainObject {
    @SerializedName("temp") var temp: Double = 0.0
    @SerializedName("temp_min") var tempMin: Double = 0.0
    @SerializedName("temp_max") var tempMax: Double = 0.0
}